settings.py

```diff
@@ -1612,3 +1612,7 @@
     pass
 else:
     initialize(DATADOG_API_KEY, DATADOG_APP_KEY)
+
+REST_FRAMEWORK = {
```

Comment:
We use the Django Rest Framework for serializing the SQL forms and cases.
This sets the default date format.